QuietPlace/microphoneListener.js

```diff
@@ -64,34 +81,69 @@ export default class MicrophoneListener extends Component {
 	//Volume threshold slider
     render() {
         return (
-        <View>
-            <Text style={styles.text}>
-			  {"\n"}Volume Threshold{"\n\n"}
-			  {this.state.value}dB
-			</Text>
-            <Slider
-              {...this.props}
-              onValueChange = {value => this.setState({value: value})}
-			  style={{width: 300, height: 30}}
-			  step = {1}
-			  minimumValue = {-100}
-			  maximumValue = {0}
-			  thumbTintColor = 'white'
-			  minimumTrackTintColor = 'pink'
-			  maximumTrackTintColor = 'white'
-            />
-        </View>
+          <View>
+              <Text style={styles.label}>
+                Volume Threshold{"\n"}
+              </Text>
+              <Text style={styles.volume}>
+                ({Math.trunc(((this.state.value + 160)/160) * 100)}%) {this.state.value} Db
+              </Text>
+              <Slider
+                {...this.props}
+                onValueChange = {value => this.setState({value: value})}
+                style={styles.slider}
+                step = {1}
+                minimumValue = {-160}
+                maximumValue = {0}
+                thumbTintColor = '#C4C4C4'
+                minimumTrackTintColor = 'black'
+                maximumTrackTintColor = 'grey'
+              />
+          
+              <TouchableOpacity style={styles.button}>
+                <Text style={styles.buttonText}>Settings</Text>
```

Comment:
I think we should look into a way of making UI elements fill the width of whatever screen it is being shown on.